src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -344,7 +344,7 @@ def getDefaultOptions(cls, parameterSetName='Default'):
             'Number of elements around': 8,
             'Number of elements through wall': 1,
             'Wall thickness': 0.016,
```

Comment:
Default parameters use the cat central path but the material wall thickness - please change to make cat parameters the default. Just need to change wall thickness to 0.007 above.

Reply:
Done.